include/isaac/defines.h

```diff
@@ -34,6 +34,18 @@
 #endif
 
 #if defined(_WIN32) || defined(_MSC_VER)
+    #define ISAACWINAPI ISAACAPI
+#else
+    #define ISAACWINAPI
+#endif
+
+#if defined(_WIN32) || defined(_MSC_VER)
+#define ISAACUNWINAPI
+#else
+#define ISAACUNWINAPI ISAACAPI
+#endif
```

Comment:
Please put a comment here to describe the usage of these two macros ISAACWINAPI and ISAACUNWINAPI. And the ISAACUNWINAPI is not a good name, how about change it to ISAACNOTWINAPI